src/request/sign-message/index.html

```diff
@@ -71,6 +73,9 @@ <h1 data-i18n="sign-msg-heading" class="nq-h1">Sign Message</h1>
 
             <div class="page-body nq-card-body">
                 <textarea id="message" readonly="readonly"></textarea>
+                <div class="tab-width">
+                    <div id="tabwidthselector"></div>
```

Comment:
Only a remark:
I liked the kebab-case we used previously throughout keyguard. Same for the translation string.